### PR TITLE
Draft release notes for Klaw documentation

### PR DESCRIPTION
**Pull request by aindriu-aiven**
Draft release notes for Klaw documentation.

Includes features and points to new documentation where possible for those new features.

### CODE REVIEW

File: docs/releases/release210.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+Redesigned Request screens with React UI
+`````````````````````````````````````````
+With the switch to React, this release features a redesign of the following user interfaces:
+
+- **Approve & Reject Topic Requests**:  Users can now approve or reject topic requests using the new UI.
+
+- **Approve & Reject Subscription Request** (Producer/Consumer): Users can now approve or reject ACL requests for both producers and consumers using the new UI.
+
+- **Approve & Reject Schema Request**: Users can now approve or reject schema requests using the new UI.
```
Comment (muralibasani) [inline docs/releases/release210.rst:23]:
```suggestion
- **Approve & Reject Schema Requests**: Users can now approve or reject schema requests using the new UI.
```

File: docs/releases/release210.rst [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+To preview the new Klaw user user interface, open the ``application.properties`` file on the Klaw **core** module, and set the value of the following property to ``true``:
+::
+    # Enable new Klaw user interface
+    klaw.coral.enabled=true
+
+.. note::
+    We are taking an incremental, feedback-driven approach in rolling out the new Klaw interfaces. By providing the feature flag to preview user interfaces, we would like you to share your valuable `feedback <https://github.com/aiven/klaw/issues/new?assignees=&labels=&template=03_feature.md>`_.
+
+Confluent Cloud
+`````````````````
```
Comment (muralibasani) [inline docs/releases/release210.rst:36]:
```suggestion
``````````````
```

File: docs/releases/release210.rst [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+Klaw now supports Confluent Cloud Topics, Acls.
+Apis implemented :
+https://docs.confluent.io/cloud/current/api.html#tag/Topic-(v3)
+https://docs.confluent.io/cloud/current/api.html#tag/ACL-(v3)
+
+For more information, see `Confluent Cloud <https://www.klaw-project.io/docs/howto/clusterconnectivity/confluent-cloud-kafka-cluster-ssl-protocol>`_.
+
+Switching Teams
+`````````````````````````````````````````````````````
```
Comment (muralibasani) [inline docs/releases/release210.rst:45]:
```suggestion
```````````````
```

File: docs/releases/release210.rst [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+::
+    # Enable new Klaw user interface
+    klaw.coral.enabled=true
+
+.. note::
+    We are taking an incremental, feedback-driven approach in rolling out the new Klaw interfaces. By providing the feature flag to preview user interfaces, we would like you to share your valuable `feedback <https://github.com/aiven/klaw/issues/new?assignees=&labels=&template=03_feature.md>`_.
+
+Confluent Cloud
+`````````````````
+Klaw now supports Confluent Cloud Topics, Acls.
```
Comment (muralibasani) [inline docs/releases/release210.rst:37]:
```suggestion
Klaw now supports Confluent Cloud Kafka Topics, Acls.
```

File: docs/releases/release210.rst [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+To preview the new Klaw user user interface, open the ``application.properties`` file on the Klaw **core** module, and set the value of the following property to ``true``:
+::
+    # Enable new Klaw user interface
+    klaw.coral.enabled=true
+
+.. note::
+    We are taking an incremental, feedback-driven approach in rolling out the new Klaw interfaces. By providing the feature flag to preview user interfaces, we would like you to share your valuable `feedback <https://github.com/aiven/klaw/issues/new?assignees=&labels=&template=03_feature.md>`_.
+
+Confluent Cloud
```
Comment (muralibasani) [inline docs/releases/release210.rst:35]:
```suggestion
Confluent Cloud Kafka
```

**Review by muralibasani — CHANGES_REQUESTED** [review]
few minor ones, rest looks good

File: docs/releases/release210.rst [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+Confluent Cloud Kafka
+Klaw now supports Confluent Cloud kafka Topics, Acls.
+Apis implemented :
+https://docs.confluent.io/cloud/current/api.html#tag/Topic-(v3)
+https://docs.confluent.io/cloud/current/api.html#tag/ACL-(v3)
+
+For more information, see `Confluent Cloud <https://www.klaw-project.io/docs/howto/clusterconnectivity/confluent-cloud-kafka-cluster-ssl-protocol>`_.
+
+Switching Teams
```
Comment (muralibasani) [inline docs/releases/release210.rst:43]:
```suggestion
Switching Teams
~~~~~~~~~~~~
```

File: docs/releases/release210.rst [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+- **Approve & Reject Topic Requests**:  Users can now approve or reject topic requests using the new UI.
+
+- **Approve & Reject Subscription Requests** (Producer/Consumer): Users can now approve or reject ACL requests for both producers and consumers using the new UI.
+
+- **Approve & Reject Schema Requests**: Users can now approve or reject schema requests using the new UI.
+
+.. image:: /../../../_static/images/topic/ApprovalRequestTopic-react.png
+
+To preview the new Klaw user user interface, open the ``application.properties`` file on the Klaw **core** module, and set the value of the following property to ``true``:
```
Comment (harshini-rangaswamy) [inline docs/releases/release210.rst:27]:
```suggestion
To preview the new Klaw user interface, open the ``application.properties`` file on the Klaw **core** module, and set the value of the following property to ``true``:
```

File: docs/releases/release210.rst [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+
+.. note::
+    We are taking an incremental, feedback-driven approach in rolling out the new Klaw interfaces. By providing the feature flag to preview user interfaces, we would like you to share your valuable `feedback <https://github.com/aiven/klaw/issues/new?assignees=&labels=&template=03_feature.md>`_.
+
+Confluent Cloud Kafka
+`````````````````````
+Klaw now supports Confluent Cloud kafka Topics, Acls.
+Apis implemented :
+https://docs.confluent.io/cloud/current/api.html#tag/Topic-(v3)
+https://docs.confluent.io/cloud/current/api.html#tag/ACL-(v3)
```
Comment (harshini-rangaswamy) [inline docs/releases/release210.rst:40]:
```suggestion
Klaw now offers Confluent Cloud Kafka Topics and ACLs support, expanding its capabilities to interact with Confluent Cloud Kafka. This release includes the implementation of the following APIs:
https://docs.confluent.io/cloud/current/api.html#tag/Topic-(v3)
https://docs.confluent.io/cloud/current/api.html#tag/ACL-(v3)
```

File: docs/releases/release210.rst (hunk 6 above)
Comment (harshini-rangaswamy) [reply docs/releases/release210.rst:44]: Suggest calling it "Switch Teams"

File: docs/releases/release210.rst [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+
+For more information, see `Confluent Cloud <https://www.klaw-project.io/docs/howto/clusterconnectivity/confluent-cloud-kafka-cluster-ssl-protocol>`_.
+
+Switching Teams
+```````````````
+With this release, users now have the ability to be a member of multiple teams and switch between those teams without logging out and logging back in. Allowing a single user to have proper auditable capabilities while also working across multiple teams.
```
Comment (harshini-rangaswamy) [inline docs/releases/release210.rst:46]:
```suggestion
Klaw now allows users to be members of multiple teams and easily switch between them without needing to log out and back in. With this new functionality, a single user can now have proper auditable capabilities while working across multiple teams. Additionally, users will benefit from increased flexibility and efficiency when working within the Klaw platform, particularly when creating or approving requests.
```

**Review by harshini-rangaswamy — CHANGES_REQUESTED** [review]
Minor suggestions.

File: docs/releases/release210.rst [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+With the switch to React, this release features a redesign of the following user interfaces:
+
+- **Approve & Reject Topic Requests**:  Users can now approve or reject topic requests using the new UI.
+
+- **Approve & Reject Subscription Requests** (Producer/Consumer): Users can now approve or reject ACL requests for both producers and consumers using the new UI.
+
+- **Approve & Reject Schema Requests**: Users can now approve or reject schema requests using the new UI.
+
+.. image:: /../../../_static/images/topic/ApprovalRequestTopic-react.png
+
```
Comment (aindriu-aiven) [inline docs/releases/release210.rst:26]: This image is missing and is needed.

**Review by harshini-rangaswamy — APPROVED** [review]
LGTM!

**Review by muralibasani — APPROVED** [review]
LGTM

**Pull request merged**